apps/dog-food/Main.hs

```diff
@@ -130,6 +128,25 @@ main = do
   env <- newSMPAgentEnv cfg {dbFile = dbFileName}
   dogFoodChat t ct env
 
+welcomeGetOpts :: IO ChatOpts
+welcomeGetOpts = do
+  appDir <- getAppUserDataDirectory "simplex"
+  opts@ChatOpts {dbFileName, termMode} <- getChatOpts appDir
+  putStrLn "simpleX chat prototype"
+  putStrLn $ "db: " <> dbFileName
+  when (os == "mingw32") $ windowsWarning termMode
+  putStrLn "type \"/help\" for usage information"
+  pure opts
+
+windowsWarning :: TermMode -> IO ()
+windowsWarning = \case
+  m@TermModeBasic -> do
+    putStrLn $ "running in Windows (terminal mode is " <> termModeName m <> ", no utf8 support)"
+    putStrLn "it is recommended to use Windows Subsystem for Linux (WSL)"
```

Comment:
nitpicking, but wouldn't it be more conventional to put explanation of abbreviation in brackets, or remove abbreviation altogether? i.e. it's not a legal document where you reuse this term later on

`WSL (Windows Subsystem for Linux)`